crates/subspace-farmer/src/plot.rs

```diff
@@ -184,6 +184,42 @@ pub struct Plot {
 }
 
 impl Plot {
+    /// Helper function for ignoring the error that given file/directory does not exist.
+    fn try_remove<P: AsRef<Path>>(
+        path: P,
+        remove: impl FnOnce(P) -> io::Result<()>,
+    ) -> io::Result<()> {
+        if path.as_ref().exists() {
+            remove(path)?;
+        }
+        Ok(())
+    }
+
+    // TODO: Remove with the next snapshot (as it is unused by now)
+    /// Erases plot in specific directory
+    pub fn erase(path: impl AsRef<Path>) -> io::Result<()> {
```

Comment:
This isn't called from anywhere thought, right?

Reply:
Yeah, I just expose it so that it could be used outside (in case)

Reply:
It must be used until next snapshot is released, so that old plots are wiped properly after upgrade